dp/cloud/go/services/dp/storage/db/table.go

```diff
@@ -47,17 +46,13 @@ func addColumns(builder sqorc.CreateTableBuilder, fields []*Field) sqorc.CreateT
 		if field.Unique {
 			builder = builder.Unique(field.Name)
 		}
-	}
-	return builder
-}
-
-func addRelations(builder sqorc.CreateTableBuilder, metadata *ModelMetadata) sqorc.CreateTableBuilder {
-	for table, column := range metadata.Relations {
-		builder = builder.ForeignKey(
-			table,
-			map[string]string{column: "id"},
-			sqorc.ColumnOnDeleteCascade,
-		)
+		if field.Relation != "" {
+			builder = builder.ForeignKey(
+				field.Relation,
+				map[string]string{field.Name: "id"},
+				sqorc.ColumnOnDeleteCascade,
```

Comment:
is this the only type of action we're expecting on delete or do we want to make it customizable?

Reply:
This is actually just used in tests, so having it hardcoded like this is fine I think